src/components/article-description.astro

```diff
@@ -0,0 +1,10 @@
+---
+interface Props {
+  text: string;
```

Comment:
you can pass the data through the props, but finally yo need to use the react component